ksqldb-common/src/main/java/io/confluent/ksql/util/BytesUtils.java

```diff
@@ -110,6 +110,42 @@ public static String encode(final byte[] value, final String encoding) {
     return Arrays.copyOfRange(getByteArray(buffer), start, end);
   }
 
+  public static int indexOf(final byte[] array, final byte[] target, final int fromIndex) {
+    for (int i = fromIndex; i < array.length; i++) {
+      if (array[i] == target[0]) {
+        if (arrayEquals(array, i, target, 0, target.length)) {
+          return i;
+        }
+      }
+    }
+
+    return -1;
+  }
+
+  @SuppressWarnings("ParameterName")
+  private static boolean arrayEquals(
```

Comment:
There's an existing `Array.equals(...)` method that only works in Java 9. The code does not build unless I add set Java version to 9 in the pom.xml.

Reply:
There's one where you specify the indexes that's Java 9 - https://docs.oracle.com/javase/9/docs/api/java/util/Arrays.html#equals-boolean:A-int-int-boolean:A-int-int-